From bc145da60ff66a6769a561de28d635861d6e7dfc Mon Sep 17 00:00:00 2001
From: Satya Durga Srinivasu Prabhala <satyap@codeaurora.org>
Date: Mon, 1 Oct 2012 23:57:42 +0530
Subject: [PATCH] display: gralloc: Add dependency on sanitized headers

libmemalloc module sources using sanitized headers,
so, adding dependency on sanitized headers to make sure
the headers gets generated before compiling the sources.

Change-Id: Ic9cad38bc690716c6c92116c7fbc6732449f6ddd
---
 libgralloc/Android.mk |   13 +++++++------
 1 files changed, 7 insertions(+), 6 deletions(-)

diff --git a/libgralloc/Android.mk b/libgralloc/Android.mk
index 1e3fc3e..fe29eb6 100644
--- a/libgralloc/Android.mk
+++ b/libgralloc/Android.mk
@@ -30,10 +30,11 @@ include $(BUILD_SHARED_LIBRARY)
 
 #MemAlloc Library
 include $(CLEAR_VARS)
-LOCAL_MODULE           := libmemalloc
-LOCAL_MODULE_TAGS      := optional
-LOCAL_C_INCLUDES       := $(common_includes) $(kernel_includes)
-LOCAL_SHARED_LIBRARIES := $(common_libs) libgenlock libqdutils
-LOCAL_CFLAGS           := $(common_flags) -DLOG_TAG=\"memalloc\"
-LOCAL_SRC_FILES        :=  ionalloc.cpp alloc_controller.cpp
+LOCAL_MODULE                  := libmemalloc
+LOCAL_MODULE_TAGS             := optional
+LOCAL_C_INCLUDES              := $(common_includes) $(kernel_includes)
+LOCAL_SHARED_LIBRARIES        := $(common_libs) libgenlock libqdutils
+LOCAL_CFLAGS                  := $(common_flags) -DLOG_TAG=\"memalloc\"
+LOCAL_ADDITIONAL_DEPENDENCIES := $(common_deps)
+LOCAL_SRC_FILES               :=  ionalloc.cpp alloc_controller.cpp
 include $(BUILD_SHARED_LIBRARY)
-- 
1.7.6

